client/src/pages/AddTour.js

```diff
@@ -32,6 +33,32 @@ export default function AddTour() {
 
   const concurrentTour = query.get("type") === "concurrent";
 
+  useEffect(() => {
+    if (id) {
+      const doFetch = async () => {
+        try {
+          const data = await getDataByID({
+            collectionName: "tasks",
+            id: id,
+          });
+          setName(data.name);
+          setStart(data.start);
+          setDest(data.dest);
+          setDate(data.date);
+          setAssignment(data.assignment);
+          setCargo(data.cargo);
+          setPriority(data.priority);
+          setInfo(data.info);
+          setCheckboxes(data.checkboxes);
+          setWeekDays(data.weekDays);
```

Comment:
Consider reducing all these states to a single state if you always set all of them the same time.
```suggestion
          setTask(task);
```

Reply:
Changes will be done tomorrow. But thanks for the advice 😄 

Opened #115 for that